tutorials/io/plot_30_reading_fnirs_data.py

```diff
@@ -74,3 +74,56 @@
              recommended.
 
 """  # noqa:E501
+
+
+
+
```

Comment:
```
Error: ./tutorials/io/plot_30_reading_fnirs_data.py:81:1: E303 too many blank lines (4)
```
```suggestion

###############################################################################
```
But really you could make this first part a part of the block-comment above.

Reply:
I was thinking we could probably make a separate tutorial, I'm currently trying to find an OctaMon and Brite 23 sample dataset that could be shared publicly.

Reply:
Something like: "How to set an fNIRS montage"

Reply:
Setting a montage for fNIRS is exactly the same as for other modalities. Why does it need its own example? I suggest it does not. Instead we should point people to the existing tutorial on setting montages?

Reply:
We discussed in person. Marking as resolved.